source/preprocessor.tex

```diff
@@ -621,6 +621,7 @@
 \pnum
 If the header identified by the \grammarterm{header-name}
 denotes an importable header\iref{module.import},
+it is implementation-defined whether
```

Comment:
`implementation-defined` should be replaced with an `\impldef`.  Perhaps `\impldef{replacing preprocessing directives with \grammarterm{preprocessing-token}{s} for an importable header}`?

Reply:
Good catch.

Reply:
Fixed.